### ISSUE TITLE
Add OPDS 2.0 support

### ISSUE THREAD

**Issue by link2xt** [melmothx/amusewiki#277]
See https://drafts.opds.io/opds-2.0

Maybe it is not ready yet, but there is a https://readium.org/ project that has [OPDS parsers](https://readium.org/architecture/projects) in Swift and Kotlin, as well as corresponding iOS and Android test applcations.

**Comment by melmothx** [comment]
Makes sense to add, still unclear how it is going to be supported by consuming applications.
